CMakeLists.txt

```diff
@@ -244,24 +208,37 @@ add_folder(ringmesh_src ringmesh_include tetrahedralize)
 add_folder(ringmesh_src ringmesh_include visualize)
 
 # Add the RINGMesh target as a shared library
-add_library(RINGMesh SHARED
-    ${ringmesh_include}
-    ${ringmesh_src})
+add_library(RINGMesh SHARED ${ringmesh_include} ${ringmesh_src})
 
 #------------------------------------------------------------------------------------------------
 # Build configuration
-# Organize outputs in a Linux way
 set_target_properties(RINGMesh PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
 set_target_properties(RINGMesh PROPERTIES ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
 
 # Add include directories of RINGMesh
 target_include_directories(RINGMesh PUBLIC ${PROJECT_SOURCE_DIR}/include)
 
-# RINGMesh depends on Geogram Tinyxml2 and zlib
-add_dependencies(RINGMesh geogram_ext tinyxml2_ext zlib_ext minizip_ext)
+# RINGMesh depends on Geogram Tinyxml2 Minizip and zlib
+list(APPEND CMAKE_MODULE_PATH ${GEOGRAM_INSTALL_PREFIX}/lib/cmake/modules)
+set(TMP_GEOGRAM_INSTALL_PREFIX ${GEOGRAM_INSTALL_PREFIX})
+set(VORPALINE_BUILD_DYNAMIC TRUE CACHE BOOL "")
+find_package(Geogram REQUIRED)
```

Comment:
Only this line should be required but thanks to geogram we have a lot of stuff and two modifications inside geogram.